index.html

```diff
@@ -0,0 +1,322 @@
+                <li class="navbarmenuli">
+                  <a href="#contact" class="navbarmenulink">Contact</a>
+                </li>
+              </ul>
+            </nav> 
+            <nav class="mobile-version">
+                <img class="menu" src="images/menu.png" alt="Menu">        
+            </nav>       
+    </header>
+    <main>
+        <section class="works">
+         <div class="headline">
+            <div class="headline-box">
+              <h2 class="myself">
+                  I’m Jorum<br/>Glad to see you!</h2>
+                  
+              <p class="details">
+                I’m a software developer! I can help you build a product , feature
+                or website Look through some of my work and experience! If you like
+                what you see and have a project you need coded, don’t hestiate to
+                contact me.
+              </p>
+              <p class="social-link-heading">LET’S CONNECT</p>
+              <div class="sociol-logo">
+                <ul class="connect-logo">
+                    <li>
+                        <a href="#"><img src="images/twitterIcon.svg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/linkedInIcon.png" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/mIcon.svg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/gitHubIcon.svg" alt=""></a>
+                    </li>
+                    <li>
+                        <a href="#"><img src="images/peaceIcon.svg" alt=""></a>
+                    </li>
+                </ul>
+            </div>
+            </div>
+        </div>       
+        </section>
+        <section class="works">
+            <div class="works-detail">            
+            <h2 class="main-topic">Tonic</h2><br>
+            <div class="works-intro">
+                <h2>Canopy</h2>
+                <ul>
+                    <li>
+                        <img src="images/Counter.png" alt="Counter"></li>
+                        <li><img src="images/backend.png" alt=""></li>
+                    
+                    <li>
+                        <img src="images/Counter.png" alt="counter"></li>
+                        <li><img src="images/2015.png" alt=""></li>                    
+                </ul>
+            </div>
+                <br>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="language">
+                    <div class="lang-icon">
+                    <ul>
+                        <li>
+                            <img src="images/html1.png" alt="html">
+                        </li>
+                        <li>
+                            <img src="images/css1.png" alt="css">
+                        </li>
```

Comment:
- [ ] As mentioned before, please add appropriate margin between cards project to match Figma design.

| Your design  | Figma design|
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/53402600/173188519-dd999acd-5e21-486a-9dd4-cff468d59b58.png)  | ![image](https://user-images.githubusercontent.com/53402600/173188537-acc5a7d7-924f-4c36-a549-6ac0b7b6111e.png) |